addons/api/addon/serializers/credential-library.js

```diff
@@ -31,9 +32,39 @@ export default class CredentialLibrarySerializer extends ApplicationSerializer {
         return super.serialize(...arguments);
     }
   }
+  serializeAttribute(snapshot, json) {
+    const value = super.serializeAttribute(...arguments);
+    const { credential_type, type, credential_mapping_overrides, isNew } =
+      snapshot.record;
+    if (
+      type === TYPE_CREDENTIAL_LIBRARY_VAULT_GENERIC &&
+      credential_mapping_overrides &&
+      credential_type &&
+      !isNew
+    ) {
+      //API expects to send null to fields if it is undefined or deleted
```

Comment:
```suggestion
      // API expects to send null to fields if it is undefined or deleted
```

Can we add a space between the slashes for comments 😅

Reply:
oops I somehow always forget to add spaces there 😿